cwl/src/main/scala/cwl/Decoder.scala

```diff
@@ -37,7 +37,12 @@ object CwlDecoder {
   }
 
   def parseJson(json: String): Parse[Cwl] =
-    EitherT{IO{CwlCodecs.decodeCwl(json).leftMap(_.getMessage).leftMap(NonEmptyList.one)} }
+    EitherT{IO{
+      CwlCodecs.decodeCwl(json).
+        leftMap{t =>
+          NonEmptyList.of(t.getMessage, t.getStackTrace.mkString("\n"))
```

Comment:
ToL: shouldn't we keep the the actual exception at this point instead ?

Reply:
Well we are working within the `Parse` type as defined as:
`EitherT[IO, NonEmptyList[String], A]`

I suppose we could go w/ exceptions in the `Nel` type.  Need to think about it a little more.

Reply:
Yeah this could also be punted as part of the "harmonize monads" tech debt.
I just think that whatever we end up choosing it should leave the possibility for keeping an exception as is when it makes sense, without having to "convert" it to an ErrorOr.